frontend/src/scenes/data-warehouse/editor/sidebar/QueryInfo.tsx

```diff
@@ -22,10 +22,6 @@ const OPTIONS = [
         value: 'never' as OrNever,
         label: ' No resync',
     },
-    {
-        value: '5min' as DataWarehouseSyncInterval,
-        label: ' Resync every 5 mins',
-    },
```

Comment:
Doing this reduces the value of materialization - knowing data could be 30 mins delayed isn't great. Any reason we're not going down the approach of using temporal schedules to throttle this instead? e.g. using `SchedulePolicy(overlap=ScheduleOverlapPolicy.SKIP)` ?

Reply:
We're actually already using the `SKIP` policy - so is there any need to remove this?

Reply:
I suppose the only reason would be to drop temporal worker usage, but I can remove this if we think it's ok.

Reply:
Yeah, I think it's absolutely fine tbh. We're pretty successful with 5-min schedules in the warehouse pipeline